### DISCUSSION TITLE
⭐️ Xcode 15.2

### DISCUSSION THREAD

**Discussion by mikhail-tokarev** [codemagic-ci-cd/codemagic-docs#2287] [Announcements]
We have released a new build machine image with Xcode 15.2 Beta 1 available. This image comes with macOS Sonoma 14.2 and has all the latest SDKs and runtimes including visionOS installed. You can use Xcode 15.2 by specifying `15.2` or `edge` as the Xcode version in your build configuration.

See more details about the build machine specs and preinstalled software in our [documentation](https://docs.codemagic.io/specs/versions-macos/).

**Comment by helinanever** [comment]
The image now contains the latest release version of Xcode 15.2.
